api/Performance.json

```diff
@@ -393,9 +400,16 @@
                 "version_removed": "3.0"
               }
             ],
-            "webview_android": {
-              "version_added": true
-            }
+            "webview_android": [
+              {
+                "version_added": "≤37"
+              },
+              {
+                "version_added": true,
```

Comment:
This too doesn't seem that great. Just skipping the prefixed entries for this until we have better data to mirror seems best to me.